update.py

```diff
@@ -44,15 +44,19 @@ def lambda_handler(event, context):
         s3.Bucket(AV_DEFINITION_S3_BUCKET).download_file(s3_path, local_path)
         print("Downloading definition file %s complete!" % (local_path))
 
-    clamav.update_defs_from_freshclam(AV_DEFINITION_PATH, CLAMAVLIB_PATH)
+    retVal = clamav.update_defs_from_freshclam(AV_DEFINITION_PATH, CLAMAVLIB_PATH)
+    if retVal != 0:
+        raise RuntimeError("clamAV update process returned %d" % (retVal))
     # If main.cvd gets updated (very rare), we will need to force freshclam
     # to download the compressed version to keep file sizes down.
     # The existence of main.cud is the trigger to know this has happened.
     if os.path.exists(os.path.join(AV_DEFINITION_PATH, "main.cud")):
         os.remove(os.path.join(AV_DEFINITION_PATH, "main.cud"))
         if os.path.exists(os.path.join(AV_DEFINITION_PATH, "main.cvd")):
             os.remove(os.path.join(AV_DEFINITION_PATH, "main.cvd"))
-        clamav.update_defs_from_freshclam(AV_DEFINITION_PATH, CLAMAVLIB_PATH)
+        retVal = clamav.update_defs_from_freshclam(AV_DEFINITION_PATH, CLAMAVLIB_PATH)
+        if retVal != 0:
+            raise RuntimeError("clamAV update process returned %d" % (retVal))
```

Comment:
It might be good to include a note that this was the `main.cvd` refresh attempt.
```suggestion
            raise RuntimeError("Refresh clamAV update process returned %d" % (retVal))
```